src/Credentials.ts

```diff
@@ -838,10 +875,33 @@ class Credentials {
    *  @return   {Promise<Object, Error>}                        a promise which resolves with a parsed response or rejects with an error.
    */
   async verifyDisclosure(token: string) {
-    const { payload, doc } = await verifyJWT(token, { audience: this.did })
+    const { payload, doc } = await verifyJWT(token, { resolver: this.resolver, audience: this.did })
     return this.processDisclosurePayload({ payload, doc })
   }
 
+  createPresentationRequest(params: PresentationRequestParams, expiresIn = 600) {
```

Comment:
`PresentationRequestParams` is supposed to be a simplified version of `DisclosureRequestPayload`, right?

@simonas-notcat does this comply with the current expectations of daf / daf-mobile?
Does daf-mobile answer `shareReq` with Presentations already or is it expecting a specific `type` in the payload?
Please weigh in here, I want to make sure we're not forking the specs here.